README.md

```diff
@@ -102,10 +102,11 @@ In the `ViewControler.swift` file, update the variables at the top of this file
     let kGraphURI = "<Your backend API>" // This is your backend API that you've configured to accept your app's tokens
     let kScopes: [String] = ["<Your backend API>/demo.read"] // This is a scope that you've configured your backend API to look for.
 ```
-
-
-
-
+> [!NOTE]
+>developers using the [Azure China Environment](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud), MUST use <your-tenant-name>.b2clogin.cn) authority, instead of `login.chinacloudapi.cn`.
+>
+> In order to use <your-tenant-name>.b2clogin.*, you will need to `setValidateAuthority(false)`. Learn more about using [b2clogin](https://docs.microsoft.com/en-us/azure/active-directory-b2c/b2clogin).

```

Comment:
We changed this in the latest MSAL 0.4.0 release. validateAuthority flag is now deprecated and you need to pass your authority into the "knownAuthorities" array in the MSALPublicClientApplicationConfig to support this